### PR TITLE
Clean up windows setup module

### PR DESCRIPTION
**Pull request by dagwieers**
##### ISSUE TYPE

 - Cleanup Pull Request

##### COMPONENT NAME

setyp.ps1

##### ANSIBLE VERSION

v2.3

##### SUMMARY

The following items were implemented:
- Replace PSObjects with normal hashes
- Replace Set-Attr function with normal hash assignment
- Re-indent code (make sure everything uses 4 spaces)
- Remove trailing semi-colons

### CODE REVIEW

File: lib/ansible/modules/windows/setup.ps1 [hunk 1]
```diff
@@ -88,142 +90,135 @@ for ($i=1; $i -le ($win32_cpu.NumberOfLogicalProcessors / $win32_cs.NumberOfProc
...
 }
-Set-Attr $result.ansible_facts "ansible_env" $env_vars
 
-$psversion = $PSVersionTable.PSVersion.Major
-Set-Attr $result.ansible_facts "ansible_powershell_version" $psversion
+$ansible_facts = @{
+    ansible_architecture = $win32_os.OSArchitecture
+    ansible_bios_date = $win32_bios.ReleaseDate.ToString("MM/dd/yyyy")
+    ansible_bios_version = $win32_bios.SMBIOSBIOSVersion
+    ansible_date_time $date
```
Comment (jhawkesworth) [inline lib/ansible/modules/windows/setup.ps1]: From the CI output, it looks like you need
`ansible_date_time = $date `
on L 137

**Comment by jhawkesworth** [comment]
Thanks for doing this.  Looks like there's a missing assignment on L 137 tripping it up at the moment unfortunately, but otherwise this is a nice cleanup.

File: lib/ansible/modules/windows/setup.ps1 (hunk 1 above)
Comment (dagwieers) [reply lib/ansible/modules/windows/setup.ps1]: Heh, I am doing a larger rewrite of Windows modules, so I was going to get back at the output of all the PRs I plan to do when I am finished. You beat me to it :-)

**Comment by jhawkesworth** [comment]
looks good now.
shipit

**Comment by dagwieers** [comment]
ready_for_review

**Comment by jborean93** [comment]
looks good.
shipit

**Pull request merged**